src/pages/episode/sections/show-notes/person-notes.js

```diff
@@ -23,15 +23,13 @@ function PersonNotes({person}) {
       {
         nothing ?
           <p>No links, tips, or picks this week</p> :
-        (
```

Comment:
Sure :) I can re-add the parentheses but they need to be 1 line up (so the rule will be fulfilled)

```
      {
        nothing ?
          <p>No links, tips, or picks this week</p> : (
          <div className={css(styles.notesContainer)}>
            {
              notesSections.map((section, index) => (
                <Notes {...section} key={index} />
              ))
            }
          </div>
        )
      }
```

Does that work for you?